### PR TITLE
Adds a js linter to frontend to enforce code style.

### PR DESCRIPTION
**Pull request by embrown**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -17,13 +17,23 @@
...
+    "karma-sinon": "^1.0.5",
+    "mocha": "^3.0.2",
+    "sinon": "^1.17.5",
     "webpack": "^1.12.11",
     "webpack-dev-server": "^1.14.1"
   },
   "scripts": {
-    "server": "webpack-dev-server --inline --colors --progress --display-error-details --display-cached --port 3000  --content-base src/ --output-public-path=/dist/",
-    "build": "webpack --inline --colors --progress --display-error-details --display-cached --content-base src/"
+    "build": "webpack --inline --colors --progress --display-error-details --display-cached --content-base src/",
```
Comment (mattben) [inline frontend/package.json]: Is the server tag no longer needed?

File: frontend/package.json [hunk 2]
```diff
@@ -17,13 +17,23 @@
...
+    "lint": "$(npm bin)/eslint -f table src/"
   },
   "license": "",
-  "repository": "http://github.com/UV-CDAT/vcdat"
+  "repository": "http://github.com/UV-CDAT/vcdat",
+  "README": ""
```
Comment (mattben) [inline frontend/package.json]: https://github.com/UV-CDAT/vcdat/blob/master/README.md

File: frontend/package.json (hunk 1 above)
Comment (embrown) [inline frontend/package.json]: @chaosphere2112 indicated that it was no longer needed in our chat the other day.

File: frontend/package.json (hunk 2 above)
Comment (embrown) [inline frontend/package.json]: I tried adding the path to vCDAT's README, but npm was still giving me errors. I'll try again.

**Comment by mattben** [comment]
Please updated the repo README. 

* fix markdown in step 6
* add step 7 about linting frontend, with link to your README

**Pull request merged**